sormas-api/src/main/java/de/symeda/sormas/api/event/EventIndexDto.java

```diff
@@ -36,6 +36,8 @@
 	public static final String PARTICIPANT_COUNT = "participantCount";
 	public static final String CASE_COUNT = "caseCount";
 	public static final String DEATH_COUNT = "deathCount";
+	public static final String CONTACT_COUNT = "contactCount"; // number of contacts whose person is involved in this event
```

Comment:
Move the comments to the actual field definitions (as JavaDoc comments, not in-line comments)